pyresample/geometry.py

```diff
@@ -1026,6 +1087,31 @@ def freeze(self, lonslats=None, resolution=None, shape=None, proj_info=None):
           the shape of the resulting area.
         proj_info:
           complementing parameters to the projection info.
+        antimeridian_mode:
+            How to handle lon/lat data crossing the anti-meridian of the
+            projection. This currently only effects lon/lat geographic
+            projections and data cases not covering the north or south pole.
```

Comment:
```suggestion
            projections and data cases not covering the prime meridian, so
            it should not cover the north or south pole.
            
```

Reply:
I'm not sure I understand the intention with your comment here. The prime meridian isn't really considered when deciding whether or not this keyword is applied.

Reply:
From my understanding, the problem with data covering the pole stems from all longitudes converging there, so data covers the pole spans the full range of possible longitudes.  There can also be data that stem all longitudes without covering the pole, if we have a satellite in an orbit that's neither polar nor geosynchronous.  For example, the TRMM orbit in this image: https://www.researchgate.net/figure/One-TRMM-swath-intersecting-path-of-Hurricane-Irene-in-2005_fig3_221206961. 

I thought the situation here is that we can handle the antimeridian as long as we have a limited span of longitudes.  More specifically, I understood that we should not have data at ±180° longitude compared to the antimeridian, otherwise we're just recreating the problem we were trying to solve.  Therefore, we can handle the antimeridian as long as data are not covering the prime meridian, correct?

(also, "effects" should be "affects" here)

Reply:
Try refreshing your page, the "affects" change should be there already.

As for the rest of that, at least for what is being checked currently in the code, the 3 things being checked are:

1. Is the CRS geographic (lon/lat)?
2. Are the X min/max greater than 355 degrees apart?
3. Is the Y point beyond +/- -89.9 degrees?

If all of these are true then we enter into the antimeridian handling. So to modify the docstring this way would be incorrect as far as what is actually being checked. If this was saying when `antimeridian_mode` should be expected to work then I think this would be more accurate, but right now it says when it takes effect (affect?). I don't think there is an easy way to check if the data crosses the prime meridian and the antimeridian without adding more calculations. You are correct that data that is already spanning all longitudes but not the poles would result in the antimeridian handling being triggered. By default it would result in a new definition from 0 to 360 longitude. That behavior existed before this PR, but I'm not sure how best to fix it either.

Reply:
OK, it's complicated :)  Never mind my suggested change then.

Reply:
Thanks for letting me take the easy road :wink: 

I think this could be cleared up in the future, but it is also a special case that may need some real world examples to be the most accurate.